### ISSUE TITLE
Form customization can't hide just the content field

### ISSUE THREAD

**Issue by Jako** [modxcms/revolution#15711]
### Summary
When modx-resource-content is disabled the whole section goes away. So it is not possible to hide just the content field and move template variables to modx-resource-content.

### Step to reproduce
Disable modx-resource-content in form customization and move template variables into modx-resource-content. The section is invisible.

### Expected behavior
There should be a way just to hide the content field and move TVs into that section.

### Environment
MODX 2.8.2

**Comment by SnowCreative** [comment]
Yes, please!!

**Comment by smg6511** [comment]
There's actually not a bug here, but the way the content fields are grouped in the Manager/Form Customization panel is misleading; _modx-resource-content_ is not an actual region, but rather a grouping of fields that are part of the _modx-resource-main-left_ region (which is where the fields [content, symlink, weblink] should be listed to avoid confusion). So, simply hide your content field and assign the tv(s) you want to move to the _modx-resource-main-left_ region.

This works on my 2.8.3 and 3.0.0-alpha installs.

**Comment by SnowCreative** [comment]
> So, simply hide your content field and assign the tv(s) you want to move to the _modx-resource-main-left_ region.

That doesn't do the trick. That will put the TVs in the left column in the top panel. What we are asking for is the ability to use the full-width panel the Content field sits in for TVs, without having the Content field there. "modx-resource-content" *is* a region; you can put TVs there, and they will then show up full-width under the Content field.

**Comment by smg6511** [comment]
OK, for 2.x I stand corrected, at least to a point. In 3.x, where I was looking more closely, the structure is a bit different and the solution above does what you'd want it to. I'd have to take a closer look at the 2.x code, but arguably modx-resource-content may not have been meant to be functionally a region, as is implied by it not showing in the Regions tab's listing in the Manager/Form Customization panel. I still say it's not a _bug_ for that reason, probably more of a feature request.

Let me ask you this though: Is that extra width really needed for a one-column group of TVs? If you could group TVs into multiple columns I could see the benefit of the full width of the panel more so.

That said, I've always thought the holy grail of customization would be the ability to move _any_ field into any region. No doubt a bigger ask ;-)

**Comment by SnowCreative** [comment]
> Let me ask you this though: Is that extra width really needed for a one-column group of TVs? If you could group TVs into multiple columns I could see the benefit of the full width of the panel more so.

Yes, in the case where the TVs are rich text fields that may have a lot of content in them. I've created sites that need quite a few rich-text TVs, and they would be much harder to edit when confined to a smaller column.

As for MODX3, there is an issue filed because the manager customization doesn't work properly (you can't put fields underneath the Content field any more). I suspect the system will get updated to work better in MODX3. 

And yes, I totally agree that the very best thing would be to be able to put any field in any panel or tab. I'm hoping some day that becomes a reality.
